test-automation/requirements.txt

```diff
@@ -0,0 +1,181 @@
+alabaster==0.7.10
+aloe==0.1.19
+aloe-webdriver==0.5.11
+anaconda-client==1.6.5
```

Comment:
Make sure that you are freezing the right environment. This seems to be your global Python environment. Create a fresh environment with `python3 -m venv env`, activate the environment with `source env/bin/activate` and then run `pip install aloe && pip freeze > requirements.txt`

Reply:
Yes. I did follow the same steps, and  generated this file. I have not pushed the env folder as it was creating a cofig with local paths

Reply:
The requirements.txt should not include anaconda-client and similar packages. I think your global Python environment is getting added here.

Reply:
Aloe seems to have only a few dependencies:
https://github.com/aloetesting/aloe_webdriver/blob/master/requirements.txt